kectl/app/cmd/cloud/init.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2019 The Kubeedge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/kubeedge/kubeedge/kectl/app/cmd/options"
+	"github.com/kubeedge/kubeedge/kectl/app/cmd/util"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	cloudInitLongDescription = `
+kectl init command bootstraps KubeEdge's cloud component.
+It checks if the pre-requisites are installed already,
+If not installed, this command will help in download,
+install and execute on the host.
+`
+	cloudInitExample = `
+kectl cloud init  --docker-version= --kubernetes-version=<version> --kubeedge-version<version> --docker-version=<version>
```

Comment:
docker-version is appearing twice. please correct.

Reply:
--kubeedge-version<version>  -> --kubeedge-version=<version>

Reply:
ok